index.html

```diff
@@ -0,0 +1,833 @@
+
+          <button class="navbar-toggler border-0" type="button" data-bs-toggle="collapse" data-bs-target="#mobile-navbar" aria-controls="mobile-navbar" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse justify-content-end" id="mobile-navbar">
+            <ul class="navbar-nav flex-row gap-4 d-flex flex-column flex-lg-row">
+              <li class="nav-item">
+                <a class="nav-link link-light" href="#project-list">Portfolio</a>
+              </li>
+
+              <li class="nav-item">
+                <a class="nav-link link-light" href="#about-me">About</a>
+              </li>
+            
+              <li class="nav-item">
+                <a class="nav-link link" href="#contact-me">Contact</a>
+              </li>
+            
+              <li class="nav-item d-none d-lg-inline-block">
+                <a class="nav-link img-fluid" href="mailto:ghooryahya@gmail.com">
+                  <img class="pb-1" src="images/icons/mail-icon.svg" alt="Mail Icon" />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="container-fluid bg-dark hero-background">
+      <div class="row row-cols-2">
+        <aside class="col-1 d-flex justify-content-center align-items-center">
+          <ul class="remove-list-styles d-none d-lg-block mt-4">
+            <li class="mb-3 mt-5">
+              <a href="#">
+                <img src="images/icons/github-icon-light.svg" alt="Github Profile" />
+              </a>
+            </li>
+  
+            <li class="mb-3">
+              <a href="#">
+                <img src="images/icons/linkedin-icon-light.svg" alt="LinkedIn Profile" />
+              </a>
+            </li>
+  
+            <li class="mb-3">
+              <a href="#">
+                <img
+                  src="images/icons/angellist-icon-light.svg"
+                  alt="Angellist Profile"
+                />
+              </a>
+            </li>
+  
+            <li class="mb-3">
+              <a href="#">
+                <img src="images/icons/twitter-icon-light.svg" alt="Twitter Profile" />
+              </a>
+            </li>
+    
+            <li class="mb-3">
+              <a href="#">
+                <img src="images/icons/medium-icon-light.svg" alt="Medium Profile" />
+              </a>
+            </li>
+          </ul>
+        </aside>
+  
+        <section class="col-11 d-flex flex-column pt-5">
+          <h1 class="text-warning display-3 fw-normal mt-5">Hey There.</h1> 
+
+          <h2 class="text-warning display-3 fw-normal">I'm Yahya</h2>
+  
+          <h2 class="text-white fw-normal">I am a Software Developer</h2>
+  
+          <p class="col-9 col-sm-7 col-md-6 col-lg-5 col-xl-4 col-xxl-3 text-white">
```

Comment:
It seems like you have lots of column changes, could you have used a percentage or something else?

Reply:
There are a few others places i saw you did the same thing